ifrs17-template/Files/TransactionalData/Actuals_CH_2020_12.csv

```diff
@@ -1,55 +1,55 @@
-﻿@@Main,,,,
-ReportingNode,Year,Month,,
-CH,2020,12,,
-@@Actual,,,,
-DataNode,AocType,ValueType,AccidentYear,Value
-DT1.1,CF,PR,,400
-DT1.1,CF,NIC,,-280
-DT1.1,CF,ACA,,-10
-DT1.1,CF,AEA,,-5
-DT1.2,CF,NIC,2020,-280
-DT2.1,CF,PR,,400
-DT2.1,CF,NIC,,-280
-DT2.1,CF,ACA,,-10
-DT2.1,CF,AEA,,-5
-DT1.3,CF,PR,,400
-DT1.3,CF,NIC,,-280
-DT1.3,CF,ACA,,-10
-DT1.3,CF,AEA,,-5
-DT1.4,CF,PR,,400
-DT1.4,CF,NIC,,-280
-DT1.4,CF,ACA,,-10
-DT1.4,CF,AEA,,-5
-DT1.4,CF,PayablePR,,15
-DT1.4,CF,ReceivablePR,,-10
-DT1.4,CF,ReceivableNIC,,-5
-DT1.4,CF,PayableNIC,,8
-DT2.2,CF,NIC,2020,-280
-DTR1.1,CF,PR,,-200
-DTR1.1,CF,NIC,,140
-DTR1.2,CF,NIC,2020,140
-DTR2.1,CF,PR,,-200
-DTR2.1,CF,NIC,,140
-DTR2.2,CF,NIC,2020,140
-DT3.1,CF,PR,,400
-DT3.1,CF,NIC,,-280
-DT3.1,CF,ACA,,-10
-DT3.1,CF,AEA,,-5
-DT4.1,CF,PR,,400
-DT4.1,CF,NIC,,-280
-DT4.1,CF,ACA,,-10
-DT4.1,CF,AEA,,-5
-DT1.5,CF,PR,,400
-DT1.5,CF,NIC,,-280
-DT1.5,CF,ACA,,-10
-DT1.5,CF,AEA,,-5
-DT1.5,CF,ReceivablePR,,-10
-DTR1.3,CF,PR,,-200
-DTR1.3,CF,NIC,,140
-DTR1.3,CF,RiPayablePR,,7.5
-DTR1.3,CF,RiReceivablePR,,-5
-DTR1.3,CF,RiReceivableNIC,,-2.5
-DTR1.3,CF,RiPayableNIC,,4
-DTR1.4,CF,PR,,-200
-DTR1.4,CF,NIC,,140
-DTR1.4,CF,RiReceivableNIC,,-10
+﻿@@Main,,,,,,,
+ReportingNode,Year,Month,,,,,
+CH,2020,12,,,,,
+@@Actual,,,,,,,
+DataNode,AocType,AmountType,EstimateType,AccidentYear,Value,,
+DT1.1,CF,PR,A,,400,,
+DT1.1,CF,NIC,A,,-280,,
+DT1.1,CF,ACA,A,,-10,,
+DT1.1,CF,AEA,A,,-5,,
+DT1.2,CF,NIC,A,2020,-280,,
+DT2.1,CF,PR,A,,400,,
+DT2.1,CF,NIC,A,,-280,,
+DT2.1,CF,ACA,A,,-10,,
+DT2.1,CF,AEA,A,,-5,,
+DT1.3,CF,PR,A,,400,,
+DT1.3,CF,NIC,A,,-280,,
+DT1.3,CF,ACA,A,,-10,,
+DT1.3,CF,AEA,A,,-5,,
+DT1.4,CF,PR,A,,400,,
+DT1.4,CF,NIC,A,,-280,,
+DT1.4,CF,ACA,A,,-10,,
+DT1.4,CF,AEA,A,,-5,,
+DT1.4,CF,PR,AA,,15,,
+DT1.4,CF,PR,OA,,-10,,
+DT1.4,CF,NIC,AA,,-5,,
+DT1.4,CF,NIC,OA,,8,,
+DT2.2,CF,NIC,A,2020,-280,,
+DTR1.1,CF,PR,A,,-200,,
+DTR1.1,CF,NIC,A,,140,,
+DTR1.2,CF,NIC,A,2020,140,,
+DTR2.1,CF,PR,A,,-200,,
+DTR2.1,CF,NIC,A,,140,,
+DTR2.2,CF,NIC,A,2020,140,,
+DT3.1,CF,PR,A,,400,,
+DT3.1,CF,NIC,A,,-280,,
+DT3.1,CF,ACA,A,,-10,,
+DT3.1,CF,AEA,A,,-5,,
+DT4.1,CF,PR,A,,400,,
+DT4.1,CF,NIC,A,,-280,,
+DT4.1,CF,ACA,A,,-10,,
+DT4.1,CF,AEA,A,,-5,,
+DT1.5,CF,PR,A,,400,,
+DT1.5,CF,NIC,A,,-280,,
+DT1.5,CF,ACA,A,,-10,,
+DT1.5,CF,AEA,A,,-5,,
+DT1.5,CF,PR,OA,,-10,,OA
+DTR1.3,CF,PR,A,,-200,,
+DTR1.3,CF,NIC,A,,140,,
+DTR1.3,CF,PR,OA,,7.5,,AA
```

Comment:
What is this extra "AA"?

Reply:
Also other cases

Reply:
my bad, I added these 2 by mistake, now canceled them. Thanks a lot!

Reply:
done